### ISSUE TITLE
Typo in the link text

### ISSUE THREAD

**Issue by anupama-pathirage** [ballerina-platform/ballerina-dev-website#543]
**Description:**
Typo in the left navigation bar of the learn pages
<img width="206" alt="image" src="https://user-images.githubusercontent.com/16300038/81556518-112a6000-93a8-11ea-9164-e02555b12629.png">

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, Browser, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by praneesha** [comment]
This is fixed via [1].

[1] https://github.com/ballerina-platform/ballerina-dev-website/pull/537/commits/ad086c04242a876d384f563807225a4df011410a

**Comment by anupama-pathirage** [comment]
@praneesha Don't we have the practice of sending PRs to this repo? Also having at least one single review?

**Comment by praneesha** [comment]
> @praneesha Don't we have the practice of sending PRs to this repo? Also having at least one single review?

This fix was already incorporated in a commit another PR[1] (as it was a minor typo) and the review will happen in that. That's why I closed this mentioning that respective commit.

[1] https://github.com/ballerina-platform/ballerina-dev-website/pull/537/commits

**Comment by anupama-pathirage** [comment]
Ok. Got it. Thanks

**Comment by anupama-pathirage** [comment]
@praneesha Shall we close this issue once the PR is merged? It will get closed automatically once the PR is merged as the issue is mentioned in the PR description.

**Comment by praneesha** [comment]
Closing as the PR is merged now.

**Comment by praneesha** [comment]
Reopening as this hasnt been applied in the prod sync and needs to be fixed again.

**Comment by praneesha** [comment]
This is fixed in [1] now.

[1] https://ballerina.io/learn/installing-ballerina/
